website/docs/components/form/primitives/partials/guidelines/guidelines.md

```diff
@@ -22,7 +22,7 @@ Use a character count to communicate the current length of the value in an input
 
 Depending on which property (or properties) are passed to the component, a different default message will be displayed by the component to communicate the relationship between the current length of the input value (`currentLength`) and the maximum length (`maxLength`), minimum length (`minLength`), or both.
 
-<video width="100%" controls loop>
+<video width="75%" controls loop>
```

Comment:
Did you mean to change this in the character count docs?

Reply:
Oh no, mistake on my end